src/App.tsx

```diff
@@ -1,51 +1,98 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
 import './App.scss';
+import './Goods.scss';
 
-// import goodsFromServer from './goods';
-
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
-
-        <button
-          type="button"
-          className="App__clear"
-        >
-          Clear
-        </button>
-      </header>
-
-      <ul>
-        <li className="Good">Dumplings</li>
-        <li className="Good">Carrot</li>
-        <li className="Good">Eggs</li>
-        <li className="Good">Ice cream</li>
-        <li className="Good">Apple</li>
-        <li className="Good">Bread</li>
-        <li className="Good">Fish</li>
-        <li className="Good">Honey</li>
-        <li className="Good Good--active">Jam</li>
-        <li className="Good">Garlic</li>
-      </ul>
-
-      {/* Put required buttons into each Good */}
-      <button
-        type="button"
-        className="Good__remove"
-      >
-        Remove
-      </button>
-
-      <button
-        type="button"
-        className="Good__select"
-      >
-        Select
-      </button>
-    </main>
-  );
+import goodsFromServer from './goods';
+
+type State = {
+  selectedGoods: string[];
 };
+
+export class App extends Component<{}, State> {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  clearSelected = () => this.setState({ selectedGoods: [] });
+
+  removeGood = (good: string) => {
+    this.setState((prevState) => (
+      { selectedGoods: prevState.selectedGoods.filter(el => el !== good) }
```

Comment:
Don't create two-letter named variables
```suggestion
      { selectedGoods: prevState.selectedGoods.filter(el => el !== good) }
```

Reply:
Linter provoking me🤪 I fixed it